.github/workflows/test-wheel-linux.yml

```diff
@@ -180,6 +180,23 @@ jobs:
           host-platform: ${{ inputs.host-platform }}
           cuda-version: ${{ inputs.cuda-version }}
 
+      - name: Set up compute-sanitizer
+        run: |
+          if [[ "${{ inputs.python-version }}" == "3.12" && "${{ inputs.local-ctk }}" == 1 ]]; then
```

Comment:
So this works on all 3 platforms? Nice!

Reply:
The compute-sanitizer tool is part of the CTK, so it is available on Windows and multiple linux flavours, yes. In this PR, I have only set it up to run on Linux pipelines, however.

Reply:
ah ok, I forgot I made this file Linux-only, nvm 😓